src/main/java/core/basesyntax/strategy/FruitAmountCalculateStrategy.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.strategy;
+
+public interface FruitAmountCalculateStrategy {
```

Comment:
шit's not a strategy, it's a handler
```suggestion
public interface OperationHandler {
```